### ISSUE TITLE
How to select language

### ISSUE THREAD

**Issue by FabricioCabeca** [igaster/laravel_cities#22]
Hi there ! I've noticed that there are many translations in the alternames field, but it is unclear how to pick one. Is the only solution to join AlternateNamesV2.zip from geonames.org or am I missing something ?

**Comment by karnaukhm** [comment]
in our case we were overriding alternative and had "lang" : "name" json pulling data from RESTful API of genomes.org keeping throttling in the mind and running a scheduled job.

**Comment by brunobg** [comment]
@FabricioCabeca did you find a good solution in the end?

**Comment by FabricioCabeca** [comment]
@brunobg nope, but you may try the approaches suggested above and use one that better fits your needs.
